.github/SECURITY.md

```diff
@@ -54,8 +54,8 @@ Versions of macOS:
 
 | macOS Version | Supported          |
 | ------------- | ------------------ |
-| 12            | :white_check_mark: |
-| 11 and lower  | :red_circle:       |
+| 14            | :white_check_mark: |
+| 12 and lower  | :red_circle:       |
```

Comment:
Should this be 13? 👀

Reply:
It should probably work, so 13 version falls into ‘unspecified’ category.

Reply:
To my understanding, anything not tested in CI is "unsupported"

Reply:
But yeah probably just leave it